doc/classes/Callable.xml

```diff
@@ -67,6 +67,7 @@
 			<return type="Callable">
 			</return>
 			<description>
+			Returns a copy of this [Callable] with the arguments bound. Bound arguments are passed after the supplied by [method call].
```

Comment:
Should have one more level of indentation (running `godot --doctool .` should fix it).

Reply:
It's not so clear so "are passed after the supplied by `call`" - the supplied what?

Reply:
arguements... Was perhaps a little late

Reply:
Still needs an extra level of indentation.